### PR TITLE
REST troubleshooting guide for Exchanges

### LINKED ISSUE #662: REST troubleshooting guide for Exchanges
Exchanges use REST directly instead of the SDK, so the SDK guide created in #654 is not very useful for them
Create a "troubleshooting" (instead of a full guide) for Exchanges using REST.
Besides most of the point covered in #654, pay attention to:
- [x] The query string to pull transactions (both standalone and embedded) from height x
- [x] Construct transfer transaction with correct deadline
- [x] Same with maxFee. 
- [x] Resolve alias using statement or hardcode both mosaicId / namespaceId

### PR DESCRIPTION
**Pull request by segfaultxavi**
Fixes #662

### CODE REVIEW

File: source/guides/exchanges/exchange-REST-troubleshooting.rst [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+       &embedded=true
+
+This query returns the list of all **transfer** transactions sent to address ``TAOXU...``, that happened between block height **70000** and **80000**, including transactions **embedded** inside other transactions.
+
+The next section explains how to parse the resulting list.
+
+Parsing of embedded transactions
+================================
+
+As shown in the `/transactions/confirmed <https://docs.symbolplatform.com/symbol-openapi/v1.0.1/#operation/searchConfirmedTransactions>`__ endpoint documentation, a successfull query returns **a data array** including transactions and metadata. Each one of the returned transactions can match a different schema depending on the transaction's type, so the ``type`` field of each transaction must be checked.
```
Comment (rg911) [inline source/guides/exchanges/exchange-REST-troubleshooting.rst:59]: The transaction endpoint actually returns a set of paginated transactions which should be mentioned here I think. If the pagination parms are not provided, default values (page size / number) will be used instead.

File: source/guides/exchanges/exchange-REST-troubleshooting.rst [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+To help choose the right amount, the `/network/fees/transaction <https://docs.symbolplatform.com/symbol-openapi/v1.0.1/#operation/getTransactionFees>`__ endpoint provides some statistics regarding the effective fees paid by the last 60 blocks. The returned data is:
+
+.. csv-table::
+   :header: "Property", "Meaning"
+   :widths: 18 82
+   :delim: ;
+
+   Highest; Biggest fee multiplier used in the last 60 blocks.
+   Median; The `median value <https://en.wikipedia.org/wiki/Median>`__ of the fee multipliers used in the last 60 blocks.
```
Comment (rg911) [inline source/guides/exchanges/exchange-REST-troubleshooting.rst:209]: Zero median fees calculated will be replaced by the defaultDynamicFeeMultiplier, this has been included in the latest rest (v2.3.6)

**Review by rg911 — CHANGES_REQUESTED** [review]

File: source/guides/exchanges/exchange-REST-troubleshooting.rst (hunk 2 above)
Comment (segfaultxavi) [reply source/guides/exchanges/exchange-REST-troubleshooting.rst]: Added a note!

File: source/guides/exchanges/exchange-REST-troubleshooting.rst (hunk 1 above)
Comment (segfaultxavi) [reply source/guides/exchanges/exchange-REST-troubleshooting.rst]: Added a note!

**Review by rg911 — APPROVED** [review]

**Review by rg911 — APPROVED** [review]

**Review by rg911 — APPROVED** [review]

**Comment by segfaultxavi** [comment]
How does the note look like?

**Pull request merged**